image/cleanup.sh

```diff
@@ -9,7 +9,7 @@ rm -rf /tmp/* /var/tmp/*
 rm -rf /var/lib/apt/lists/*
 
 # clean up python bytecode
-find / -name *.pyc -delete
```

Comment:
I remember getting plenty of "permission denied" errors here as `find` was trying to traverse into `/proc` and `/sys`folders.